sensor_msgs/Image image
bool loss
int16 oh_sc_i
int16 oh_lc_i
int16 oh_s_i
int16 oh_l_i
---
float32[1728] return_img
float32[10] sc
float32[10] lc
float32[13] s
float32[36] l
float32 r_loss
